content/projects/gsoc/2024/project-ideas.html.haml

```diff
@@ -0,0 +1,161 @@
+---
+layout: simplepage
+title: "GSoC 2024 Project Ideas"
+year: 2024
+tags:
+- gsoc
+- gsoc2024
+project: gsoc
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+//The individual page's layout is located at /content/_layouts/gsocprojectidea.html.haml
+
+:css
+  .pi_table {
+    width: 100%;
+  }
+  .pi_table_header {
+    text-align: center;
+  }
+  .pi_table_category {
+    width: 100px;
+  }
+  .pi_table_skills {
+    width: 175px;
+  }
+
+%img{:title => "Jenkins GSoC", :src => expand_link("/images/gsoc/jenkins-gsoc-logo_small.png"), :style => "float:right", :width => "128"}
+
+
+%p
+  = succeed '.' do
+    This page aggregates project ideas for Google Summer of Code
+    = page.year
+  = succeed '.' do
+    See more information about this project and applications
+    %a{:href => expand_link("/projects/gsoc/")}
+      on the Jenkins Google Summer of Code page
+%p
+  Below you can find project ideas which have been proposed for this year.
+  New ideas may be proposed by interested mentors or GSoC contributors (e.g. new features in the core, "write a plugin for MY_TOOL_OR_SERVICE", etc.).
+  Project ideas without potential mentors will be considered though applicants may need to work with the community and GSoC org admins to find mentors.
```

Comment:
```suggestion
  New ideas may be proposed by interested mentors or GSoC contributors, such as new features in the core or "write a plugin for MY_TOOL_OR_SERVICE".
  Project ideas without potential mentors will be considered, though applicants may need to work with the community and GSoC org admins to find mentors.
```
Suggestion for grammar/syntax, and removing `e.g.` as this is not recommended in the [Microsoft styleguide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg).